index.bs

```diff
@@ -147,11 +145,80 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
 
 <p>In [[!ISOBMFF]] or [[!HEIF]] files, if an [=AV1 Alpha Image Item=] (respectively. an [=AV1 Alpha Image Sequence=]) is used, the <code>aux_type</code> field of the <code>AuxiliaryTypeProperty</code> (respectively <code>AuxiliaryTypeInfoBox</code>) SHALL be set to <code>urn:mpeg:mpegB:cicp:systems:auxiliary:alpha</code>, as defined in [[!MIAF]].</p>
 
-<h2 id="AVIF-baseline-profile">MIAF AV1 Baseline Profile</h2>
+  <h2 id="brands-and-file-extensions">Brands and file extensions</h2>
+
+  <h3 id="AVIF-general-brand">AVIF General Brand</h3>
+
+Files conformant with the brand-independent restrictions in this document shall include the brand
+<dfn value="" export="" for="AVIF General brand">avif</dfn> in the [=compatible_brands=] field of the [=FileTypeBox=].
+  Files shall also carry a compatible brand to identify the AVIF profile, if any, with which the file complies.
+
+  <h3 id="image-and-image-collection-files">AV1 image and image collection files</h3>
+  The file extension ".avif" and the MIME Content Type "image/avif" should be used for AVIF files that satisfy the following requirements:
+  - The file does not contain tracks, including image sequences, of any kind.
+  - The [=primary image=] is either an [=AV1 Image Item=] or a derived image that references one or more coded images that all are [=AV1 Image Items=].
+  - The AVIF file conforms to at least one AVIF profile, and the profile is listed in the [=compatible_brands=] field of the [=FileTypeBox=].
+  - The AVIF file specifies 'avif' in the [=major_brand=] field of the [=FileTypeBox=].
```

Comment:
Why? A file can be conformant to the brand without having it as a major brand.

Reply:
Yes, you are right that it can. However, this text is appearing in the section that defines the rules for the ".avif" file extension and since this is a new file extension we have the opportunity to button down the rules a little bit and require that 'avif' be specified if as the major_brand if you want to use the .avif file extension. The problem with the ISOBMFF file type box is that it has variable size (can contain up to 32 different brands, I think) and this makes it difficult for implementations that try to recognize file types by looking for a well-known byte pattern at a well-known offset in the file.
There is a risk that if most, but not all, implementations put 'avif' in the major_brand box, that some clients will expect that 'avif' will always appear in the major_brand box. Those implementations might break if someone creates a .avif file with 'avif' only in the compatible_brands box. This potential interop problem is avoided by defining stricter rules right now, and require that 'avif' be in the major_brand box.

Reply:
> can contain up to 32 different brands, I think

No. It's unlimited.

I'm a bit uneasy as this goes beyond the practice in the file format and HEIF/MIAF. 

For example, HEIF says the following about the 'mif1'/'msf1' brands (A.2.1.1/A.3.1.1):
> When this brand is the major brand, the defined file extension and MIME type should be used.

Also all MIME type definitions (e.g. Annex D) indicate:
> The file extension and MIME type of a file in this family usually reflect the major brand in the FileTypeBox.

Also, MIAF says:
> A MIAF file shall use the filename extensions specified by HEIF to identify the presence of specific image coding formats, as summarized in Table 2.

This does not seem to leave room for defining new extensions. Maybe @dwsinger can comment here.

Reply:
It does leave room for defining new extensions. They simply go in the compatible_brands field, as usual. The quote from the MIME type definitions support the idea that the file extension should reflect the major brand. I.e., if the file extension is ".avif" then the major brand is supposed to be 'avif', and vice versa.

Reply:
> It does leave room for defining new extensions.

How? I was talking about "file extension". Table 2 says for any other codec than AVC or HEVC you must use ".heif" or ".heifs". Although it's confusing as the title of table says informative ...



> The quote from the MIME type definitions support the idea that the file extension should reflect the major brand. I.e., if the file extension is ".avif" then the major brand is supposed to be 'avif', and vice versa.

Yes, I'm beginning to think so.

Reply:
MIAF is specifically about those conformance points. We might need to ask MPEG to make it clear that external codecs can be structurally-MIAF-compatible, and what compatible brand(s) should be used.

Please don't ever 'claim' the major brand; you can say "if the major brand is X, the extension shall be Y" and so on.  (I think you know this).